server/apps/distributed-app/docs/modules/ROOT/pages/benchmark/james-benchmark.adoc

```diff
@@ -0,0 +1,95 @@
+= Distributed James Server benchmark
+:navtitle: James benchmarks
+
+This document provides benchmark methodology and basic performance of Distributed James as a basis for a James administrator who
+can test and evaluate if his Distributed James is performing well.
+
+It includes:
+
+* A sample Distributed James deployment topology
+* Propose benchmark methodology and base performance result. This aims to help operators quickly identify
+performance issues.
+
+== Sample deployment topology
+
+We deploy a sample topology of Distributed James with these following components:
+
+- Apache Cassandra 4 as main database
+- OpenDistro 1.13.1 as search engine
+- RabbitMQ 3.8.17 as message queue
+- OVH Swift S3 as an object storage
```

Comment:
Please specofy servers counts, CPU, memory. Specify also the counts of James PODs, CPU, memory.